aci.go

```diff
@@ -897,6 +896,12 @@ func (p *ACIProvider) RunInContainer(ctx context.Context, namespace, name, conta
 	return ctx.Err()
 }
 
+// ConfigureNode enables a provider to configure the node object that
+// will be used for Kubernetes.
+func (p *ACIProvider) ConfigureNode(ctx context.Context, node *v1.Node) {
+	// noop
```

Comment:
This should replace these:

https://github.com/virtual-kubelet/azure-aci/blob/c0a9805483acac9fcc0bca59152da242bd30064a/aci.go#L953-L1040